types/wellknown/index.d.ts

```diff
@@ -1,7 +1,32 @@
 // Type definitions for wellknown 0.5
 // Project: https://github.com/mapbox/wellknown#readme
-// Definitions by: Yair Tawil <https://github.com/yairtawil>
+// Definitions by: Davide Scalzo <https://github.com/davodesign84>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function parse(input: string): {};
-export function stringify(gj: {}): string;
+export type GeoJSONPosition = [number, number] | [number, number, number];
+export interface Geometry<T, C> {
+    type: T;
+    coordinates: C;
+}
+
+export type GeoJSONPoint = Geometry<'Point', GeoJSONPosition>;
+export type GeoJSONMultiPoint = Geometry<'MultiPoint', GeoJSONPosition[]>;
+
+export type GeoJSONLineString = Geometry<'LineString', GeoJSONPosition[]>;
+
+export type GeoJSONMultiLineString = Geometry<'MultiLineString', GeoJSONPosition[][]>;
+
+export type GeoJSONPolygon = Geometry<'Polygon', GeoJSONPosition[][]>;
+
+export type GeoJSONMultiPolygon = Geometry<'MultiPolygon', GeoJSONPosition[][][]>;
+
+export type GeoJSONGeometry =
+    | GeoJSONPoint
+    | GeoJSONMultiPoint
+    | GeoJSONLineString
+    | GeoJSONMultiLineString
+    | GeoJSONPolygon
+    | GeoJSONMultiPolygon;
+
+export function parse(input: string): GeoJSONGeometry;
```

Comment:
So the documentation states it also returns `null`, when parse fails:
https://github.com/mapbox/wellknown#parsewkt

Reply:
ah great spot!